redis_sessions/session.py

```diff
@@ -210,7 +225,33 @@ def get_real_stored_key(self, session_key):
         else:
             session_key = str(session_key)
 
-        prefix = settings.SESSION_REDIS_PREFIX
+        if self.conf == 'alt_settings':
+            prefix = alt_settings.SESSION_REDIS_PREFIX
+        else:
+            prefix = default_settings.SESSION_REDIS_PREFIX
         if not prefix:
             return session_key
         return ':'.join([prefix, session_key])
+
+    def get_conf(self, session_key):
+        session_existence_check = (not default_settings.DROP_ORIGINAL_SESSION_STORE and default_settings.SESSION_STORE_MIGRATION_MODE)
+        no_session_existence_check = (default_settings.DROP_ORIGINAL_SESSION_STORE and default_settings.SESSION_STORE_MIGRATION_MODE)
+        if session_existence_check:
+            # check for session existence in the current store
+            logger.info("checking if session key exists in default store")
+            self.server = self.get_redis_server(session_key, default_settings)
+            if self.exists(session_key):
+                logger.info("using default settings 1")
```

Comment:
The numbers in the loggers, can we have some sort of distinctive log event that can be used to track how this is moving along?

Reply:
You cant keep track of this in logs in a clear way

Reply:
Like say I want to see how the migration is moving along what event do I track